source/_integrations/roborock.markdown

```diff
@@ -142,7 +142,7 @@ No. This integration requires information from your Roborock app to set up and u
 As of right now - no. When the vacuum is disconnected from the internet, it will attempt to disconnect itself from Wi-Fi and reconnect itself until it can reach the Roborock servers. We are looking for the best way to handle this and see what can be blocked while still allowing the vacuum to function. 
 
 ### What devices are supported?
-If you can add your device to the Roborock app - it is supported. However, some older vacuums like the Roborock S5 must be connected using the Mi Home app and can be set up in Home Assistant through the [Xiaomi Miio](/integrations/xiaomi_miio/) integration.
+Most devices that can be added to the Roborock app can be added, you can check [here](https://python-roborock.readthedocs.io/en/latest/supported_devices.html) to see what devices are supported. Some older vacuums like the Roborock S5 must be connected using the Mi Home app and can be set up in Home Assistant through the [Xiaomi Miio](/integrations/xiaomi_miio/) integration. Some devices have not yet been reverse engineered.
```

Comment:
We should not send users into libraries. That is an abstraction they shouldn't care about from an end-user perspective.

Let's include it here instead.